seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/ConfigBuilder.java

```diff
@@ -116,23 +117,39 @@ public static Config of(
         }
         log.info(
                 "Parsed config file: \n{}",
-                mapToString(configDesensitization(config.root().unwrapped())));
+                mapToString(
+                        configDesensitization(
+                                config.root().unwrapped(),
+                                ConfigShadeUtils.getSensitiveOptions(config))));
         return config;
     }
 
-    public static Map<String, Object> configDesensitization(Map<String, Object> configMap) {
+    public static Map<String, Object> configDesensitization(
+            Map<String, Object> configMap, Set<String> sensitiveKeywords) {
         return configMap.entrySet().stream()
                 .collect(
                         LinkedHashMap::new,
                         (m, p) -> {
                             String key = p.getKey();
                             Object value = p.getValue();
-                            if (Arrays.asList(DEFAULT_SENSITIVE_KEYWORDS)
-                                    .contains(key.toLowerCase())) {
-                                m.put(key, "******");
+                            if (sensitiveKeywords.contains(key.toLowerCase())) {
```

Comment:
Why is the `toLowerCase` needed here? As in the doc, we didn't specify whether we can match with the ignore case.

Reply:
We don't care keywords case. The old behavior already convert to lowercase https://github.com/apache/seatunnel/pull/8739/files#diff-f4fa1a9b065718cc135d232712063b7b3a9135933396dd2674253da754d95c86L131. So I think we should keep it. cc @remones